crates/ruff_python_formatter/tests/snapshots/format@expression__call.py.snap

```diff
@@ -386,10 +386,10 @@ threshold_date = datetime.datetime.now() - datetime.timedelta(  # comment
 )
 
 # Parenthesized and opening-parenthesis comments
-func(x for x in y)
+func((x for x in y))
```

Comment:
this looks worse to me

Reply:
My reasoning here is that Black (and ruff) preserve parentheses of nested expressions. Not doing so violates that principle and it is not that Ruff introduces the parentheses if the user wrote the generator without. 

So I think this comes down to the question of whether Ruff/Black should normalize all parenthesized expressions. The current philosophy is to preserve them. That's why I'm leaning toward chaining it.

CC: @zanieb who discovered this during dogfooding

Reply:
I think I would be leaning towards removing them if there's precedence for e.g. tuples but black preserve tuple parentheses too: 

```python
a = (1, 2)
```

Reply:
i'm fine with merging this! It's a the clear compatibility problem, i just don't like the decision to have double parentheses `((...))` (without or without whitespace in between)

Reply:
I agree, in which case we should change `call((1, 2))` too